### PR TITLE
kojira docs

### PR DESCRIPTION
**Pull request by odra**
fixes: #37 

## Changes

* Adds kojira crd docs

## Vetification

* Run `make html` from the docs folder

### CODE REVIEW

File: docs/source/user-guide/kojira.rst [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+| cacert_secret        | koji-hub-ca-cert                   | string  |
++----------------------+------------------------------------+---------+
+| client_cert_secret   | koji-builder-client-cert           | string  |
++----------------------+------------------------------------+---------+
+
+
+image
+-----
+
+The the full qualified image name to pull kojira from.
```
Comment (Zlopez) [inline docs/source/user-guide/kojira.rst:54]:
```suggestion
The full qualified image name to pull kojira from.
```

File: docs/source/user-guide/kojira.rst [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+replicas
+--------
+
+The amount of kojira replicas to deploy.
```
Comment (Zlopez) [inline docs/source/user-guide/kojira.rst:59]:
```suggestion
The amount of kojira's replicas to deploy.
```

File: docs/source/user-guide/kojira.rst [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+hub_host
+--------
+
+Koji-hub hostname (includes port) for hub connections.
+
+src
+---
+
+Indicateds if kojira should include srpms in repos.
```
Comment (Zlopez) [inline docs/source/user-guide/kojira.rst:81]:
```suggestion
Indicates if kojira should include srpms in repos.
```

**Comment by Zlopez** [comment]
Few small thinks

**Comment by Zlopez** [comment]
Also, I'm not sure if the Makefile was updated with kojira deployment.

**Comment by odra** [comment]
it was in the initial structure pr

**Pull request merged**